common/ethnicities/wc_ethnicities_ogre.txt

```diff
@@ -383,10 +383,10 @@ ogre_ethnicity = {
 	
```

Comment:
What about limiting hairstyles to the ones that don't clip or fly?
https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/pull/188#pullrequestreview-595418862

Reply:
I personally don't think clipping is such a big deal (played too much WoW so I'm used to it) but can do. 👌

Reply:
Taken care of in 3cb6fea594db534be0e8c1ebd5ecfd4ffadc4d5f!